classes/MyTableAdmin.php

```diff
@@ -322,13 +311,13 @@ protected function outputField(array $field, $key, array $record, array $options
         return $output;
     }
 
-    /** Get all tables (and comments to them) in the database and store them to tables
+    /** Get all tables in the database (including comments) and store them to tables
      */
     public function getTables()
     {
         $this->tables = array();
         $query = $this->dbms->query('SELECT TABLE_NAME, TABLE_COMMENT FROM information_schema.TABLES '
-                . 'WHERE TABLE_SCHEMA = "' . $this->escape($this->options['database']) . '"');
+                . 'WHERE TABLE_SCHEMA = "' . $this->escapeSQL($this->options['database']) . '"');
```

Comment:
K úvaze: nebylo by jasnější používat $this->dbms->escapeSQL($string) a zbavit se wrapperu escapeSQL v MyTableAdmin?

Reply:
Ok, nebráním se tomu, ale až v nějakém dalším pull requestu.